packages/hapi-auth-udaru/bench/util/runner.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
```

Comment:
I think we should move the benchmarks into their own shared package. Otherwise they are just duplicated.

Reply:
Good point. Especially since I'm working on the fastify version. Will do it.

Reply:
Actually I was thinking...if these benchmark is run from within the entire udaru repository, why do we even need a shared package at all? Couldn't just move it to the root folder and that's it?

Reply:
we'll likely have to support both plugins for some time, anything that cuts down on duplication would be welcomed